src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
```

Comment:
It's better to have a private static final `Random` instance instead of creating a new `Random` object every time `getRandomColor()` is called. 🌟